lib/run/index.js

```diff
@@ -150,6 +151,16 @@ module.exports = function (options, callback) {
             (options.bail === true || (_.isObject(options.bail) && options.bail.failure))) ?
             stopOnFailure = true : stopOnFailure = false;
 
+        if (options.skipIteration) {
```

Comment:
The code flow is weird here. This block will proceed for a negative value. 

The code path complexity for such a tiny feature is unnecessarily high. Please attempt to streamline it.

- first validate
- then arrive at the range
- engage slicing on valid range